### DISCUSSION TITLE
Icon Request: icon-underage (adults only, 18 ban)

### DISCUSSION THREAD

**Discussion by Xarex** [FortAwesome/Font-Awesome#21194] [Icon Requests]
I run a website that has some erotic literature and bad language on some pages.. I was looking for an icon that could distinguish this. "Mature Content" or "Adults Only" would be great.

**Comment by ivan-tellez** [comment]
Your request is not a concrete object.   

Maybe could ask for the text   "18+"   in a rounded square.

http://comps.canstockphoto.com/can-stock-photo_csp15606678.jpg

**Comment by tagliala** [comment]
@ivan-tellez thanks.

:underage:

**Comment by Xarex** [comment]
The 18+ is a good icon. Thanks @ivan-tellez and thanks @tagliala.   

For temporary, I ended up using the "warning" icon.

But it is a good icon for warning visitors that they are about to read or see something that might contain adult mature content, violence, etc.

**Comment by AgentSmith0** [comment]
Any updates to this?

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
🔞 [U+1F51E](https://emojipedia.org/emoji/%F0%9F%94%9E/) No One Under Eighteen
Also known as `NSFW`.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
